syntax = "proto3";


message Position32f {
    float x = 1;
    float y = 2;
    float z = 3;
}


message Status {
    uint64 timestamp = 1;
    Position32f position = 2;
    string debug = 3;
}

